src/Apps/__stories__/Apps.story.tsx

```diff
@@ -1,13 +1,8 @@
 import { MockRouter } from "DevTools/MockRouter"
 import React from "react"
 import { storiesOf } from "storybook/storiesOf"
-import { routes as auctionRoutes } from "../Auction/routes"
+import { routes as searchRoutes } from "../Search/routes"
 
-storiesOf("Apps", module).add("Auction Registration", () => {
-  return (
-    <MockRouter
-      routes={auctionRoutes}
-      initialRoute="/auction-registration2/weekly-mocktion"
-    />
-  )
+storiesOf("Apps", module).add("Search Results", () => {
+  return <MockRouter routes={searchRoutes} initialRoute="/search?term=andy" />
```

Comment:
Confused about this diff. Auction stories here changed to Search, and AuctionApp.story.tsx now holds this?

Reply:
Might have been a bad rebase here. I'll fix this up. Thanks!